backend/src/main/java/botobo/core/infrastructure/s3/FakeS3Uploader.java

```diff
@@ -0,0 +1,24 @@
+package botobo.core.infrastructure.s3;
+
+import botobo.core.domain.user.User;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+
+@Profile({"local", "test"})
+@Slf4j
+@RequiredArgsConstructor
+@Component
```

Comment:
```suggestion
@Profile({"local", "test"})
@Component
```
사용하지 않는 코드는 지워줘도 될 것 같습니다!